embedding/embedding.py

```diff
@@ -32,10 +33,32 @@ def __init__(self, embeddings: List[List[float]], metadata: EmbeddingMetadata, t
         self.metadata = metadata
         self.text = text
 
+    def clean_text_for_vector_db(self, text):
```

Comment:
we should keep this method inside chunk and call it clean text instead of to the embedding